sway/input/keyboard.c

```diff
@@ -761,6 +802,10 @@ struct xkb_keymap *sway_keyboard_compile_keymap(struct input_config *ic,
 	xkb_context_set_user_data(context, error);
 	xkb_context_set_log_fn(context, handle_xkb_context_log);
 
+	if (getauxval(AT_SECURE) > 0) {
```

Comment:
From the man page:

> This function is a nonstandard glibc extension.

Reply:
Oh hm, that's disappointing.

Reply:
Meh, secure_getenv is itself also a GNU extension or I'd consider just trying that. I think I'll investigate if xkbcommon is willing to drop secure_getenv here or maybe provide some way for us to get around it.